src/assets/content/dictionary/java.md

```diff
@@ -1,6 +1,15 @@
 ---
-title: Java
+title: Application Programming Interface (API)
 definition: ''
 perspectives: []
 
+title: Java
+definition: Java is an object-oriented programming language that produces software for multiple platforms. When a programmer writes a Java application, the compiled code (known as bytecode) runs on most operating systems (OS), including Windows, Linux and Mac OS. Java derives much of its syntax from the C and C++ programming languages.
+sources:
+- sourceurl: [https://www.techopedia.com/definition/3927/java]
+perspectives:
+- meaning: A perspective designed for working with Java projects. It consists of an editor area and the following views:
+Declaration
```

Comment:
question: I don't quite understand this perspective. It might help to think of the perspective area as the following:
"As a <role>, <term> means <meaning>." This might be something like, "As a backend developer, Java means a language I use to code web, mobile, or desktop applications."